quickstart/pg-init/pg-init.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Do we really need this?

Reply:
this sets up two tables in docker compose 1 for hydra and 1 for kratos. I dont think you can do this in the docker-compose yaml itself without having a separate script. It also could spin up 2 entire postgres instances